PastaGA.py

```diff
@@ -0,0 +1,354 @@
+            fit_val += 10
+        if self.__gene_1 == "basil":
+            fit_val += 7
+        if self.__gene_2 == "basil":
+            fit_val += 7
+        if self.__gene_3 == "basil":
+            fit_val += 7
+        if self.__gene_4 == "basil":
+            fit_val += 7
+        if self.__gene_1 == "spinach":
+            fit_val += 10
+        if self.__gene_2 == "spinach":
+            fit_val += 10
+        if self.__gene_3 == "spinach":
+            fit_val += 10
+        if self.__gene_4 == "spinach":
+            fit_val += 10
+        if self.__gene_1 == "chives":
+            fit_val += 2
+        if self.__gene_2 == "chives":
+            fit_val += 2
+        if self.__gene_3 == "chives":
+            fit_val += 2
+        if self.__gene_4 == "chives":
+            fit_val += 2
+        if self.__gene_1 == "shrimp":
+            fit_val += 2
+        if self.__gene_2 == "shrimp":
+            fit_val += 2
+        if self.__gene_3 == "shrimp":
+            fit_val += 2
+        if self.__gene_4 == "shrimp":
+            fit_val += 2
+        if self.__gene_1 == "mushroom":
+            fit_val += 2
+        if self.__gene_2 == "mushroom":
+            fit_val += 2
+        if self.__gene_3 == "mushroom":
+            fit_val += 2
+        if self.__gene_4 == "mushroom":
+            fit_val += 2
+        if self.__gene_1 == "macaroni":
+            fit_val += 5
+        if self.__gene_2 == "macaroni":
+            fit_val += 5
+        if self.__gene_3 == "macaroni":
+            fit_val += 5
+        if self.__gene_4 == "macaroni":
+            fit_val += 5
+        if self.__gene_1 == "feta cheese":
+            fit_val += 2
+        if self.__gene_2 == "feta cheese":
+            fit_val += 2
+        if self.__gene_3 == "feta cheese":
+            fit_val += 2
+        if self.__gene_4 == "feta cheese":
+            fit_val += 2
+        if self.__gene_1 == "olives":
+            fit_val += 0
+        if self.__gene_2 == "olives":
+            fit_val += 0
+        if self.__gene_3 == "olives":
+            fit_val += 0
+        if self.__gene_4 == "olives":
+            fit_val += 0
+        if self.__gene_1 == "noodles":
+            fit_val += 7
+        if self.__gene_2 == "noodles":
+            fit_val += 7
+        if self.__gene_3 == "noodles":
+            fit_val += 7
+        if self.__gene_4 == "noodles":
+            fit_val += 7
+        if self.__gene_1 == "sausage":
+            fit_val += 7
+        if self.__gene_2 == "sausage":
+            fit_val += 7
+        if self.__gene_3 == "sausage":
+            fit_val += 7
+        if self.__gene_4 == "sausage":
+            fit_val += 7
+        self.fitvalue = fit_val
+
+    def get_fitvalue(self):
+        #gives the set fitvalue
+        return self.fitvalue
+
+class Chrom_Population:
+
+    __chrom_list = []
+    __chrom_fitdict = {}
+    __prob_dict = {}
+
+    def __init__(self, chrom_list):
+        Chrom_Population.__chrom_list = chrom_list
```

Comment:
Here you are setting a class level variable, not an instance variable. If you make multiple populations, they will clobber each other. Instead use:

self.__chrom_list = chrom_list